src/Screens/Main/Chat/Input.tsx

```diff
@@ -38,28 +44,46 @@ export default ({ buddyId }: Props) => {
     sendMessage({ buddyId, text: messageContent });
   };
 
+  const inputContainerBg = showPending ? colors.lightestGray : colors.white;
+
+  React.useEffect(() => {
+    if (isPending) {
+      let pendingTimeOut = setTimeout(() => setShowPending(true), 500);
+
+      return () => clearTimeout(pendingTimeOut);
+    } else {
+      setShowPending(false);
+    }
+  }, [isPending]);
+
   return (
     <SafeAreaView style={styles.container} forceInset={{ bottom: 'always' }}>
-      <RN.View style={styles.inputContainer}>
+      <RN.View
+        style={[styles.inputContainer, { backgroundColor: inputContainerBg }]}
+      >
         <RN.TextInput
-          style={styles.inputText}
+          style={[styles.inputText]}
           onChangeText={storeMessage}
           value={messageContent}
           multiline={true}
-          editable={true}
+          editable={isPending ? false : true}
           testID="main.chat.input.input"
         />
       </RN.View>
       <RN.TouchableOpacity
         onPress={onSend}
-        disabled={messageContent === ''}
-        style={[styles.send, { backgroundColor: color }]}
+        disabled={disableSending}
```

Comment:
imperfect naming, disable naming hints that it is a function. Here it looks like it is a value. isSendingDisabled? or sendingDisabled. would be better.